openfl/federated/__init__.py

```diff
@@ -14,3 +14,9 @@
 if pkgutil.find_loader('torch'):
     from .task import PyTorchTaskRunner, FederatedModel  # NOQA
     from .data import PyTorchDataLoader, FederatedDataSet  # NOQA
+
+__all__ = [
+    'Plan',
+    'TaskRunner',
+    'DataLoader',
+]
```

Comment:
Can we remove this change? After #998 is merged, this will become a source of merge conflict. Also #998 removes this construct.

Reply:
This was added at my request. '__all__' is necessary for the API docs to render correctly. Without this, entire pages were missing in the prior PR. Maybe it's possible to modify the configuration to avoid this though.